src/types/point.js

```diff
@@ -29,7 +29,10 @@ export default class PointAnnotation extends Element {
     ctx.fillStyle = options.backgroundColor;
     setShadowStyle(ctx, options);
     const stroke = setBorderStyle(ctx, options);
+    // REMINDER: Sets to 0 in order to avoid that Chart.js function will perform stroking
     options.borderWidth = 0;
+    // for animation on radius, the element property must be used
+    options.radius = this.radius;
```

Comment:
I had this doubt and I don't like it as well, in fact I have added a comment. Well, to maintain the radius in the animation properties, we need to copy the `drawPoint` function of CHART.JS and put in the plugin, doing the needed updates.
Doing that, we could also change it to avoid to set the `borderWidth` to 0 (for stroking). What do you think?